src/Command/RoutesCommand.php

```diff
@@ -35,14 +35,45 @@ class RoutesCommand extends Command
      */
     public function execute(Arguments $args, ConsoleIo $io): ?int
     {
-        $output = [
-            ['Route name', 'URI template', 'Defaults'],
-        ];
+        $header = ['Route name', 'URI template', 'Controller', 'Action', 'Plugin', 'Prefix', 'Method(s)'];
```

Comment:
Having `Prefix` and `Plugin` before `Controller` would be more inline with how URL parts are generally mapped.

Reply:
More exact order is `Plugin` before `Prefix`.